pkgs/desktops/deepin/library/dtkgui/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, cmake
+, qttools
+, wrapQtAppsHook
+, librsvg
+, lxqt
+, dtkcore
+, qtimageformats
+, freeimage
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dtkgui";
+  version = "5.6.3";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-r6ZwGPiK6CcKEg8RoHV07wJbQI3idJFV3WFtuKim8n4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    qttools
+    pkg-config
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    lxqt.libqtxdg
+  ];
+
+  propagatedBuildInputs = [
+    dtkcore
+    librsvg
+    qtimageformats
+    freeimage
+  ];
```

Comment:
Do we really need to do this?

Reply:
`dtkcore` is required in `dtkgui.pc`, not sure about the rest.

Reply:
qtimageformats is a runtime dependency，is uesd for parse dci icons (webp format)
![image](https://user-images.githubusercontent.com/22803888/215120641-77705c42-a479-4329-9340-f8765689c151.png)
librsvg is  runtime dependency uesd by DSvgRenderer
freeimage is  runtime dependency uesd by  DImageHandler

Reply:
This cannot be added via the qt wrapper or another variable?

Reply:
QT_PLUGIN_PATH will be set if qtimageformats in `buildInputs`  and wrapQtAppsHook in `nativeBuildInputs`
if dont use `propagatedBuildInputs`， we have to set qtimageformats wrapper for all dtk application